src/main/scala/com/rbmhtechnology/calliope/serializer/SequencedEventSerializer.scala

```diff
@@ -61,10 +64,14 @@ abstract class SequencedEventSerializer(system: ExtendedActorSystem, payloadSeri
       .setCreationTimestamp(sequencedEvent.creationTimestamp.toEpochMilli)
 
   def sequencedEvent(format: SequencedEventFormat): SequencedEvent[_] =
-    SequencedEvent(
-      payloadSerializer.payload(format.getPayload),
-      format.getSourceId,
-      format.getSequenceNr,
-      Instant.ofEpochMilli(format.getCreationTimestamp)
-    )
+    try {
+      SequencedEvent(
+        payloadSerializer.payload(format.getPayload),
+        format.getSourceId,
+        format.getSequenceNr,
+        Instant.ofEpochMilli(format.getCreationTimestamp)
+      )
+    } catch {
+      case NonFatal(err) => throw new PayloadNotSerializableException("Unable to deserialize payload", err, format.getPayload)
```

Comment:
Because the payload a different.

Given you have an Envelope with `PayloadFormat` wrapping a `SequencedEventFormat` wrapping a `AppPayload`. If you catch the exception only in `PayloadSerializer.payload` then the payload information in the Exception is about the `SequencedEventFormat` payload, catching it in the `SequencedEventSerializer` will actually result in information about the `AppPayload`.